homeassistant/components/generic/translations/en.json

```diff
@@ -42,6 +39,13 @@
                     "verify_ssl": "Verify SSL certificate"
                 },
                 "description": "Enter the settings to connect to the camera."
+            },
+            "user_confirm_still": {
+                "data": {
+                    "confirmed_ok": "This image looks good."
+                },
+                "description": "![Camera Still Image Preview](/api/camera_preview_proxy/{preview_id})",
```

Comment:
My concern would be: This could be messed with using translations

Reply:
Yes, if a translator alters the text in the round brackets.

I think this would be true for any image or link in the same context.

Is it possible to use a constant that is pulled in from somewhere else to define the link?

Perhaps I could pass the full link as a parameter rather than just the unique id...?

Reply:
This is updated now to reduce the chance of messing with it during translation.